nucypher/cli/characters/stake.py

```diff
@@ -532,18 +555,22 @@ def divide(click_config,
         STAKEHOLDER.stakes.refresh()
         current_stake = STAKEHOLDER.stakes[index]
     else:
-        current_stake = select_stake(stakeholder=STAKEHOLDER, emitter=emitter)
+        current_stake = select_stake(stakeholder=STAKEHOLDER, emitter=emitter, divisible=True)
 
     #
     # Stage Stake
     #
 
     # Value
     if not value:
-        value = click.prompt(f"Enter target value (must be less than or equal to {str(current_stake.value)})",
+        value = click.prompt(f"Enter target value "
+                             f"({NU.from_nunits(STAKEHOLDER.economics.minimum_allowed_locked)}"
+                             f" - {str(current_stake.value)})",
```

Comment:
Should the maximum end of the target value range be `max(min_allowed_locked, current_stake.value - min_allowed_locked)`?

We already know that `current_stake` is >= `2 * min_allowed_locked` from the `select_stake` call.

Reply:
@KPrasch any thoughts here ^?